src/System.Private.Uri/src/System/IPv6AddressHelper.cs

```diff
@@ -2,9 +2,8 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System.Diagnostics;
```

Comment:
nit: this line should be after the following one to keep things in alphabetical order.

Reply:
Yup